cmd/utils/flags.go

```diff
@@ -993,6 +993,23 @@ var (
 		Name:  "config",
 		Usage: "TOML configuration file",
 	}
+	BlockGenerationIntervalFlag = cli.Int64Flag{
+		Name: "block-generation-interval",
```

Comment:
How about using a dot `.` for separation for consistency with other flags?

Reply:
@jeongkyun-oh A dot is used to group flags, but I don't think `block.generation.interval` is a propper hierarchical group.

Reply:
Alright. Thanks.